src/main/resources/assets/computercraft/lua/bios.lua

```diff
@@ -817,7 +817,8 @@ end
 settings.set( "shell.allow_startup", true )
 settings.set( "shell.allow_disk_startup", (commands == nil) )
 settings.set( "shell.autocomplete", true )
-settings.set( "edit.autocomplete", true )
+settings.set( "edit.default_extension", ".lua" )
+settings.set( "edit.extension", true )
```

Comment:
Remove this setting. Extensionless behaviour should be indicated by setting default_extension to ""